Header header
float32                     origin_x
float32                     origin_y
float32                     resolution
perception/ScanPoint[]      p
int64                       counter
driving_common/Pose             robot
float64                     timestamp


